js/dom.js

```diff
@@ -14,7 +14,7 @@ for (const type of [NodeList, NamedNodeMap, HTMLCollection, HTMLAllCollection])
 {
   // display a full text tooltip on buttons with ellipsis overflow and no inherent title
   const addTooltipsToEllipsized = () => {
-    for (const btn of document.getElementsByTagName('button')) {
+    for (const btn of $$('button')) {
```

Comment:
This code is executed on resize event so it's better to keep getElementsByTagName as it's much-much-much faster than querySelectorAll used in $$.